### DISCUSSION TITLE
Does ShapeCrawler trim-safe?

### DISCUSSION THREAD

**Discussion by BiDuang** [ShapeCrawler/ShapeCrawler#697] [Q&A]
Looks like trimming the ShapeCrawler will cause `IL2104` warning. Is there a solution that can make the dll trim-safe?

**Comment by AdamShakhabov** [comment]
@BiDuang, thank you for the report.

I'm not good familiar with the trimming topic. I've created a separate issue https://github.com/ShapeCrawler/ShapeCrawler/issues/708 for this investigation. You are welcome to update its description.
